api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupTxRTT2019Report.java

```diff
@@ -123,10 +121,10 @@ public ReportDefinition constructReportDefinition() {
         CohortDefinition males = cohortDefinitionLibrary.males();
         CohortDefinition females = cohortDefinitionLibrary.females();
 
-        CohortDefinition onArtDuringQuarter = hivCohortDefinitionLibrary.getPatientsHavingRegimenDuringPeriod();
+        CohortDefinition tc_curr = hivCohortDefinitionLibrary.getActivePatientsWithLostToFollowUpAsByDays("28");
         CohortDefinition patientsWithNoClinicalContactsForAbove28DaysByBeginningOfPeriod = getPatientsWithNoClinicalContactsForAbove28DaysByBeginningOfPeriod();
```

Comment:
@dbaluku The variable patientsWithNoClinicalContactsForAbove28DaysByBeginningOfPeriod seems un-necessary

Reply:
Done